iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/optional.inl

```diff
@@ -288,6 +307,7 @@ template <typename T>
 template <typename... Targs>
 inline void optional<T>::construct_value(Targs&&... args) noexcept
 {
+    // AXIVION Next Construct AutosarC++19_03-M5.2.8 : Pointer conversion is safe here.
```

Comment:
Here I would use the same rational I provided in the comments above.

Reply:
Sure.